examples/lidar_visual/GlutWindow.cc

```diff
@@ -249,12 +249,19 @@ void updateLidarVisual()
   // change detected due to key press
   if (g_lidarVisualUpdateDirty)
   {
-    g_lidar->SetDisplayNonHitting(g_showNonHitting);
-    g_lidar->SetPoints(g_lidarData);
-    g_lidar->SetType(g_lidarVisType);
-    g_lidar->Update();
+    if(g_clear == true)
```

Comment:
https://github.com/ignitionrobotics/ign-rendering/pull/116/checks?check_run_id=922956902

```suggestion
    if (g_clear == true)
```